Source/Core/Core/HW/VideoInterface.h

```diff
@@ -400,8 +404,16 @@ class VideoInterfaceManager
 private:
   u32 GetHalfLinesPerEvenField() const;
   u32 GetHalfLinesPerOddField() const;
-  u32 GetTicksPerEvenField() const;
-  u32 GetTicksPerOddField() const;
+  template <bool ignore_overclocking = false>
```

Comment:
Rather than having a `bool` template parameter I think it would be much more clear at the call-sites if you instead had a separate function like, `GetOverclockedTicksPerEvenField`.

Reply:
I used templates because, for other functions that call `GetTicksPerHalfLine`, template parameters can be easily propagated. If I made a separate function like `GetOverclockedTicksPerHalfLine` then I would also need to make a `GetOverclockedTicksPerEvenField`.

The alternative would be to have a normal argument instead, though that's a performance hit if the compiler doesn't inline the function. Though I'm probably just paranoid, that function isn't even called frequently (and the function is definitely going to be inlined if Dolphin is compiled with LTO)...

Do I make it a normal argument instead?